.travis.yml

```diff
@@ -51,6 +51,7 @@ before_script:
     - if [ -n "$OSX_SDK" -a -f depends/sdk-sources/MacOSX${OSX_SDK}.sdk.tar.gz ]; then tar -C depends/SDKs -xf depends/sdk-sources/MacOSX${OSX_SDK}.sdk.tar.gz; fi
     - make $MAKEJOBS -C depends HOST=$HOST $DEP_OPTS
 script:
+    - if [ "$RUN_TESTS" = "true" -a "$TRAVIS_REPO_SLUG" = "bitcoin/bitcoin" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then contrib/verify-commits/verify-commits.sh; fi
```

Comment:
Are you sure this is working. I could imagine gpg needs the keys first. Something like
```sh
while read line; do travis_retry gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys $line; done < contrib/verify-commits/trusted-keys; fi
```

would do it, but it might not be enough. I tried to do this once but I gave up as it was too tricky to get right on travis...